examples/documentation/documentation.bal

```diff
@@ -30,10 +31,21 @@ public type DummyObject abstract object {
 // This is the documentation attachment of the `DummyRecord` record.
 # `DummyRecord` is a user-defined record.
 # This `DummyRecord` has a `string` data field and an `int` data field.
+# This is a reference to a function `dummyFunction`.
 #
 # + fieldOne - This is the description of the `DummyRecord`'s `fieldOne` field.
 # + fieldTwo - This is the description of the `DummyRecord`'s `fieldTwo` field.
 public type DummyRecord record {
     string fieldOne;
     int fieldTwo;
 };
+
+// This is the documentation attachment of the `dummyFunction`.
+# The function `dummyFunction` is a user-defined function with
+# + x - One input parameter.
+# + y - Another input parameter.
+# + return - the sum of input parameters.
+# This function add parameter `x` and parameter `y`.
```

Comment:
For consistency, can we update this block as 

```ballerina
# This `dummyFunction` function accepts two `int` parameters and returns the sum of the two.
#
# + x - First parameter.
# + y - Second parameter.
# + return - The sum of the two parameters.
```

Reply:
Done.